_pkgdown.yml

```diff
@@ -299,7 +313,13 @@ news:
 navbar:
   structure:
     left: [intro, reference, news, articles, extensions]
+    right: [search, github, ai]
   components:
     extensions:
-      text: Extensions
```

Comment:
![image](https://github.com/user-attachments/assets/7806e1e7-fca2-4764-a572-c062bb23165b)

There appears to be sufficient space to write this out in full on my end

Reply:
You need to make the browser a bit narrower